js/apps/admin-ui/pom.xml

```diff
@@ -136,7 +136,9 @@
       "resourceUrl": "${resourceUrl}",
       "masterRealm": "${masterRealm}",
       "resourceVersion": "${resourceVersion}",
-      "isRunningAsTheme": true
+      "isRunningAsTheme": true,
+      "logo": "${properties.logo!""}",
```

Comment:
Since this property contains a path perhaps it would be good to name it more explicitly? Perhaps `logoPath`?

Reply:
You have a point, but this is the property name that's already used in account console V2.

Reply:
It's okay to change this no? Folks will have to opt-in to the new theme anyways, especially when extending.